### PR TITLE
Display file log when test fails.

### PR DESCRIPTION
**Pull request by ldez**
### What does this PR do?

Display file log when test fails.

Clean some test methods.

### Motivation

Have more information when tests fails.

### More

- [x] Added/updated tests

### CODE REVIEW

**Comment by ldez** [comment]
@vdemeester I already tested that:
- we already use `TearDown` or `TearDownSuite` for some tests: it's not possible to have 2 methods
- this need to create a state, and that create some other problems.

File: integration/integration_test.go [hunk 1]
```diff
@@ -19,8 +19,8 @@ import (
 	checker "github.com/vdemeester/shakers"
 )
 
-var integration = flag.Bool("integration", false, "run integration tests")
-var container = flag.Bool("container", false, "run container integration tests")
+var integration = flag.Bool("integration", true, "run integration tests")
```
Comment (mmatur) [inline integration/integration_test.go]: Could you please change to `false`

File: integration/integration_test.go [hunk 2]
```diff
@@ -19,8 +19,8 @@ import (
...
+var container = flag.Bool("container", true, "run container integration tests")
```
Comment (mmatur) [inline integration/integration_test.go]: Could you please change to `false`

Comment (ldez) [reply integration/integration_test.go]: oops

**Pull request merged**